### PR TITLE
Resolve #121, improved responsiveness of the homepage. Replaced the p…

### LINKED ISSUE #121: Update responsiveness of homepage
Update responsiveness of homepage

### PR DESCRIPTION
**Pull request by MadsHeilmann**
…opovers with poppers.

Also, I messed up which branch i used for this PR - so please disregard the number and name of the branch.

### CODE REVIEW

**Review by audio-engineer — DISMISSED** [review]
Have you checked whether the content fits into the `<MainContainer>` compoment on mobile and desktop without overflow?

File: components/server/home-page.tsx [hunk 1]
```diff
@@ -1,19 +1,30 @@
...
 import CropSquareIcon from "@mui/icons-material/CropSquare";
+import Box from "@mui/material/Box";
+import HomePagePopper from "@/components/client/home-page-popper";
 
 const HomePage: FC = (): ReactElement | null => {
   return (
-    <Grid container direction="column" spacing={8}>
-      <Grid>
-        <Typography variant="h1" align="center">
+    <Box>
```
[the remark this answers is not in the document]
Comment (MadsHeilmann) [reply components/server/home-page.tsx:14]: It functions as a wrapper div

**Comment by MadsHeilmann** [comment]
> Have you checked whether the content fits into the `<MainContainer>` compoment on mobile and desktop without overflow?

It works seemlesly on mobile now, as well as desktop - without overflow and without the weird undefined behavior there was when clicking the buttons on mobile.

File: components/server/home-page.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (MadsHeilmann) [reply components/server/home-page.tsx]: Has been updated

**Review by audio-engineer — APPROVED** [review]

**Review by Danar435 — APPROVED** [review]

**Pull request merged**